docs/design/volcano-simulator.md

```diff
@@ -0,0 +1,239 @@
+# Volcano Simulator
+
+## Background
+Volcano Simulator provides a simulation platform for Volcano scheduler, with the goal of creating a realistic simulation 
+environment for evaluating the performance of Volcano scheduler. 
```

Comment:
functional correctness is also our consideration.

Reply:
In my opinio, it also can be used to run `regression testing`.

Reply:
Yeah, it is the visualization of the scheduling process and results. You can refer to https://github.com/kubernetes-sigs/kube-scheduler-simulator?tab=readme-ov-file#simulator.